testing.md

```diff
@@ -49,6 +58,12 @@ the function will return NIL in that case."
     (t nil)))
 ~~~
 
+We will test that code:
+
+- we must ensure that the content read in a file is the expected content,
+- we must test that the condition is signaled if the file doesn't exist.
```

Comment:
Sorry I don't understand this paragraph. This is what we'll test below?

If so:
```suggestion
Next, our test code shall:
- ensure that the content read in a file is the expected content,
- test that the condition is signaled if the file doesn't exist.
```

still, I'm a bit lost

Reply:
oh. mmh. "We will write tests for that code" (the code above). "In particular, we must ensure: that the content… - that a condition…".

Reply:
Much clearer :+1:  :)

Reply:
great, thanks :)